site/src/main/paradox/index.md

```diff
@@ -26,7 +26,7 @@ See @scaladoc[Scio Scaladocs](com.spotify.scio.index)  for current API documenta
 - @ref:[Runners](Runners.md) - how Scio handles Beam runners and runner specific logic
 - @ref:[Scio data guideline](Scio-data-guideline.md) - guideline for common problems
 - @ref:[Apache Beam](Apache-Beam.md) - notes on Apache Beam compatibility
-- @ref:[Changelog](Changelog.md) - breaking changes in each release
```

Comment:
do we still need `Changelog.md`?

Reply:
It's not gone, just moved into `releases/breaking-changes.md`! 👍